### PR TITLE
Refactor

### PR DESCRIPTION
**Pull request by segoranov**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,3 @@
+cp.sh
+PHPMailer-master/
+mailservice.php
```
Comment (VladimirAnaniev) [inline .gitignore:3]: Why do we need these?

File: models/CourseService.php [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+                    $row['course_title'],
+                    $row['teacher_id'],
+                    $row['course_day'],
+                    $row['course_from'],
+                    $row['course_to']
+                )
+            );
+        }
+    }
+}
```
Comment (VladimirAnaniev) [inline models/CourseService.php:30]: Nit: Shouldn't `<?php` tags be closed at the end of the file?
Same for the rest of the new files

File: models/ParkingDB.php [hunk 3]
```diff
@@ -0,0 +1,35 @@
+<?php
+
+const DB_HOST_NAME = "localhost";
+const DB_USERNAME = "root";
+const DB_PASSWORD = "";
+const DB_NAME = "parkingdb";
```
Comment (VladimirAnaniev) [inline models/ParkingDB.php]: think these constants should be in a separate config file

File: views/listcourses.php [hunk 4]
```diff
@@ -1,18 +1,19 @@
...
-    $courses = array();
+$hasCourses = false;
+$courses = array();
 
-    if (is_logged_in()) {
-        $courses = User::getCourses($_SESSION['u_id']);
-        $hasCourses = sizeof($courses) > 0;
-    }
+if (is_logged_in()) {
+    $courses = UserService::getCourses($_SESSION['u_id']);
```
Comment (VladimirAnaniev) [inline views/listcourses.php]: Should't we be calling the controller instead?

**Review by VladimirAnaniev — APPROVED** [review]
Looks good. please consider the comments above :)

File: .gitignore (hunk 1 above)
Comment (segoranov) [reply .gitignore:3]: We don't need them, that's why they are in .gitignore. I used it to facilitate my git usage. Probably shouldn't be here, but for our purposes I think that's OK :D

File: models/CourseService.php (hunk 2 above)
Comment (segoranov) [reply models/CourseService.php:30]: No, that's optional. Depends on how we agree to do it.

**Comment by VladimirAnaniev** [comment]
Side comment: looks like you are committing as `root`. You can configure git to use your github account's credentials

File: views/listcourses.php (hunk 4 above)
Comment (segoranov) [reply views/listcourses.php]: Yes, correct. Will fix that.

**Pull request merged**